pom.xml

```diff
@@ -129,59 +129,6 @@
 					</dependency>
 				</dependencies>
 			</plugin>
-			<plugin>
-				<groupId>com.github.eirslett</groupId>
-				<artifactId>frontend-maven-plugin</artifactId>
-				<!-- requires maven 3.1+ -->
-				<version>1.9.1</version>
-				<configuration>
-					<workingDirectory>src/main/web</workingDirectory>
-					<installDirectory>target</installDirectory>
-					<yarnInheritsProxyConfigFromMaven>false</yarnInheritsProxyConfigFromMaven>
-				</configuration>
-
-				<executions>
-					<execution>
-						<id>install node and yarn</id>
-						<goals>
-							<goal>install-node-and-yarn</goal>
-						</goals>
-						<configuration>
-							<nodeVersion>v12.18.3</nodeVersion>
-							<yarnVersion>v1.22.5</yarnVersion>
-						</configuration>
-					</execution>
-					<execution>
-						<id>yarn install</id>
-						<goals>
-							<goal>yarn</goal>
-						</goals>
-						<configuration>
-							<workingDirectory>src/main/web</workingDirectory>
-						</configuration>
-					</execution>
-					<execution>
-						<id>yarn clean</id>
-						<goals>
-							<goal>yarn</goal>
-						</goals>
-						<configuration>
-							<arguments>clean</arguments>
-							<workingDirectory>src/main/web</workingDirectory>
-						</configuration>
-					</execution>
-					<execution>
-						<id>yarn build</id>
-						<goals>
-							<goal>yarn</goal>
-						</goals>
-						<configuration>
-							<arguments>build</arguments>
-							<workingDirectory>src/main/web</workingDirectory>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
```

Comment:
That's weird... Did I not commit this to master already? Or did I mess something up?